heron/stmgr/src/cpp/server/stmgr-main.cpp

```diff
@@ -70,8 +70,15 @@ int main(int argc, char* argv[]) {
     LOG(FATAL) << "Corrupt topology defn file" << std::endl;
   }
 
+  sp_int64 high_watermark = heron::config::HeronInternalsConfigReader::Instance()
+                              ->GetHeronStreammgrNetworkBackpressureHighwatermarkMb() *
+                                1024 * 1024;
```

Comment:
Instead of multiplying by 1024* 1024 - every time, can you define a constant in one of the include file called 

const int MB = 1024 * 1024;

somewhere in the include file